pt/deploy/README.md

```diff
@@ -193,14 +194,12 @@ Assim como fez em seu próprio computador, você pode criar um virtualenv na Pyt
 
     20:20 ~ $ source myvenv/bin/activate
 
-    (mvenv)20:20 ~ $  pip install django whitenoise
+    (mvenv)20:20 ~ $  pip install -r requirements.txt
```

Comment:
There's a pip freeze in this translation, but that's completely missing in the EN version. In fact, in the EN version whitenoise isn't even used. Most likely because it's updated to 1.10.